docs/configuration.md

```diff
@@ -1700,6 +1700,8 @@ to use on each machine and maximum memory.
 Since `spark-env.sh` is a shell script, some of these can be set programmatically -- for example, you might
 compute `SPARK_LOCAL_IP` by looking up the IP of a specific network interface.
 
+Note: When running Spark on YARN in cluster mode, environment variables need to be set using the <code>spark.yarn.appMasterEnv.[EnvironmentVariableName]</code> property in your `conf/spark-defaults.conf` file.  Environment variables that are set in `spark-env.sh` will not be reflected in the YARN Application Master process in cluster mode.  See the [YARN-related Spark Properties](running-on-yarn.html#spark-properties) for more information.
```

Comment:
Yeah as I recall `<code>` is required in tables since it occurs within the `<table>` tag and ticks aren't parsed there. But otherwise use back ticks.